.lintr

```diff
@@ -1,45 +1,35 @@
-linters: linters_with_defaults(
-    any_duplicated_linter(),
-    any_is_na_linter(),
-    backport_linter("oldrel-4", except = c("R_user_dir", "str2lang", "str2expression", "deparse1", "...names")),
-    consecutive_assertion_linter(),
-    expect_comparison_linter(),
-    expect_identical_linter(),
-    expect_length_linter(),
-    expect_named_linter(),
-    expect_not_linter(),
-    expect_null_linter(),
-    expect_s3_class_linter(),
-    expect_s4_class_linter(),
-    expect_true_false_linter(),
-    expect_type_linter(),
-    fixed_regex_linter(),
-    for_loop_index_linter(),
-    if_not_else_linter(),
-    implicit_assignment_linter(),
-    implicit_integer_linter(),
-    keyword_quote_linter(),
-    lengths_linter(),
+linters: all_linters(
+    backport_linter("3.6.0", except = c("R_user_dir", "deparse1", "...names")),
     line_length_linter(120L),
-    missing_argument_linter(),
-    nested_ifelse_linter(),
-    numeric_leading_zero_linter(),
-    outer_negation_linter(),
-    paste_linter(),
-    redundant_equals_linter(),
-    redundant_ifelse_linter(),
-    sort_linter(),
-    sprintf_linter(),
-    strings_as_factors_linter(),
-    undesirable_function_linter(c(Sys.setenv = NA_character_, mapply = NA_character_, structure = NA_character_)),
-    unnecessary_nested_if_linter(),
-    unnecessary_lambda_linter(),
+    object_overwrite_linter(
+      allow_names = c("end", "line", "lines", "new", "offset", "pipe", "start", "symbols")
+    ),
+    undesirable_function_linter(modify_defaults(
+      defaults = default_undesirable_functions,
+      Sys.setenv = NA_character_,
+      mapply = NA_character_,
+      structure = NA_character_,
```

Comment:
While we're at it, this could use some explanation, why we don't like `structure()`.

Reply:
Oops, these are already `default_undesirable_functions`, this just served to give us a crappier lint message. Removed them.